### PR TITLE
nds2-client Update to 0.16.2 nds2-client-swig 0.16.3 new Portfile

### PR DESCRIPTION
**Pull request by emaros**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.12.6 16G1709
Xcode 9.2 9C40b

and

macOS 10.14.1 18B75
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/nds2-client-swig/Portfile [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+PortSystem        1.0
+PortGroup         cmake 1.0
+
+name              nds2-client-swig
+version           0.16.3
+revision          0
+categories        science
+platforms         darwin
+license           GPL-2
+maintainers       ligo.org:ed.maros openmaintainer
```
Comment (mf2k) [inline science/nds2-client-swig/Portfile]: Add your GitHub handle here so you automatically get notified when someone submit a PR against this port.

File: science/nds2-client/Portfile [hunk 2]
```diff
@@ -4,12 +4,12 @@ PortSystem        1.0
...
 name              nds2-client
-version           0.15.2
-revision          1
+version           0.16.2
+revision          0
 categories        science
 platforms         darwin
 license           GPL-2
-maintainers       {ligo.org:ed.maros @emaros} openmaintainer
+maintainers       ligo.org:ed.maros openmaintainer
```
Comment (mf2k) [inline science/nds2-client/Portfile]: You have removed your GitHub handle here, but it should stay in.

**Comment by mf2k** [comment]
The commit message doe snot follow the guideline. Ideally it would have been two commits, one for each port. But at least separate each port to its own line in the single commit message.

**Comment by pmetzger** [comment]
I'm not sure I love the commits being combined (which also means the commit messages are still a bit nonstandard) but it overall seems fine so I'll merge it. Thanks for the updates, @emaros!

**Comment by pmetzger** [comment]
(BTW, what I meant by that was that the new port should probably have been in a separate commit.)

**Comment by emaros** [comment]
I will make separate requests in the future. Thank you for accepting and teaching me.

**Pull request merged**